
syntax = "proto3";


message ReceiveSecret {
  string role=1;
  bool active=2;
  //THis active will be True if the user has asked for password reset,
  //Once reset, the secret will be updated again as encryption with same key
  //is not good, New aes key will be generated with new salt
  string created_on=3;
  uint32 nonce=4;
  string nonce_hash=5;
  string signed_nonce=6;
  uint32 idx =8;
  string public=9;
}
